### DISCUSSION TITLE
How to unlock keyring in i3wm

### DISCUSSION THREAD

**Discussion by s8foschm** [i3/i3#6304] [Q&A]
[Thinkpad X395, Manjaro xfce with i3wm manually installed]
I have an issue where, when I reboot my laptop and go directly into i3wm, the popup to unlock the keyring is not shown. This shouldn't be an issue, but it implies that sometimes, dialog windows (like file managers) and some notifications (like the notifications that should be sent by Spotify every time the song changes) are not shown. When I go into xfce and open e.g. VSCodium, it asks to unlock the keyring when I want to open a file. I type in my password, and the dialog shows up. When I do that immediately in i3wm, nothing happens, and it doesn't show the pop up to unlock the keyring. How can I fix this? Is there something I can install to unlock my keyring in i3wm directly?

**Comment by orestisfl** [comment]
There's an option to automatically unlock it, give https://wiki.archlinux.org/title/GNOME/Keyring a read. Otherwise, I think using `secret-tool search --unlock xxx yyy` should prompt you to unlock but I can't verify right now.
